Wildcards(Breakout Folder) (#59)
DavidXanatos/sandboxie-docs: Content/BreakoutFolder.md

   [DefaultBox]
   BreakoutFolder=C:\Downloads
   BreakoutFolder=E:\
   BreakoutFolder=C:\App\*
   BreakoutFolder=C:\App?
```

The first example specifies that any content inside the folder "C:\Downloads" will be launched unsandboxed.

Entire drives can also be specified as shown in the second example.

> The third and fourth line shows basic characters from wildcard.

Reviewer: ```suggestion
The third and fourth lines show basic characters from wildcards.
```

> `*` defines any subfolder beyond App folder(App\1, App\1\1 etc.).

Reviewer: ```suggestion
`*` defines any subfolder beyond App folder (App\1, App\1\1 and etc.).
```

> `?` defines single character from folder (App1, App2 .. App9) but not subfolders.

Reviewer: ```suggestion
`?` defines a single character from folder (App1, App2 .. App9) but not subfolders.
```

   [DefaultBox]
   BreakoutFolder=C:\Downloads
   BreakoutFolder=E:\
   BreakoutFolder=C:\App\*
   BreakoutFolder=C:\App?
   BreakoutFolder=C:\?pp\*
```

The first example specifies that any content inside the folder "C:\Downloads" will be launched unsandboxed.

Entire drives can also be specified as shown in the second example.

The third and fourth lines show basic characters from wildcards.
`*` defines any subfolder beyond App folder (App\1, App\1\1 and etc.).
`?` defines a single character from folder (App1, App2 .. App9) but not subfolders.

> Also you can combine several wildcards to match specified folder name and subfolders

Reviewer: ```suggestion
Also, you can combine several wildcards to match a specified folder name and subfolders
```
or

```suggestion
Also, you can combine several wildcards to match the specified folder name and subfolders
```

> Also, you can combine several wildcards to match the specified folder name and subfolders

Reviewer: ```suggestion
Also, you can combine several wildcards to match the specified folder name and subfolders.
```
